test/system/archive_baseline.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/sh -f
+
+echo
+
+if [ $# -ne 1 ]; then
+    echo "Invoke archive_baseline.sh -help for usage."
+    exit 1
+fi
+
+if [ $1 == "-help" ] || [ $1 == "--help" ]; then
+cat << EOF1
+NAME
+
+	archive_baseline.sh - archive pretag baselines to set locations on
+                              izumi and derecho.
+
+
+SYNOPSIS
+
+	archive_baseline.sh TAGNAME
+	  [-help]
+
+
+ENVIROMENT VARIABLES
+
+	CESM_TESTDIR - Directory that contains the CESM finished results you wish to archive.
+	CAM_FC      - Compiler used, only used on izumi (GNU,NAG), where the compiler
+                      name is appended to the archive directory.
+
+
+BASELINE ARCHIVED LOCATION
+
+	izumi:     /fs/cgd/csm/models/atm/cam/pretag_bl/TAGNAME_gnu
+	                   /fs/cgd/csm/models/atm/cam/pretag_bl/TAGNAME_nag
```

Comment:
I am still not sure if we will archive official tests on Izumi long-term, but just in case should we put them in a different location, say:

```suggestion
	izumi:     /fs/cgd/csm/models/atm/sima/pretag_bl/TAGNAME_gnu
	                   /fs/cgd/csm/models/atm/sima/pretag_bl/TAGNAME_nag
```

Reply:
Fixed all locations